htdocs/fourn/commande/dispatch.php

```diff
@@ -1290,7 +1290,7 @@
 					print '</td>';
 				}
 
-				if ($action != 'editline' ||  && $lineid != $objp->dispatchlineid)
+				if ($action != 'editline' && $lineid != $objp->dispatchlineid)
```

Comment:
???. It should be || for me.
If we are not in editline  ($action != 'editline), we must not show the Save and Cancel button.
If the line we show is not the line we are editing, we must not show the Save and Cancel button 
If one of the case is true, we must enter the if and not the else (we do not show the Save and Cancel button)
Can you provide a screenshot of what's wrong ?

Reply:
done